cpp/src/core/sroptcryst.h

```diff
@@ -52,12 +54,14 @@ class srTOptCryst : public srTGenOptElem {
 	//TVector3d m_nv; // horizontal, vertical and longitudinal coordinates of outward normal to crystal surface in the frame of incident beam
 	TVector3d m_tv; // horizontal, vertical and longitudinal coordinates of central tangential vector [m] in the frame of incident beam
 	TVector3d m_sv; // horizontal, vertical and longitudinal coordinates of central saggital vector [m] in the frame of incident beam
+	TVector3d v_nv; //* horizontal, vertical and longitudinal coordinates of outward normal to crystal surface in the frame of incident beam */ //ARF06062019 Introduce Vector3d& v_nv,
```

Comment:
Why should we add this vector to member variables, if it seems to be calculatable from m_tv, m_sv? Even if we decide to add it, why we can't call it "m_nv" - in line with m_tv, m_sv naming convention?

Reply:
This helped in debugging the code for the proper definition of 'm_HXAi'. now that the code is running 'v_nv' is not needed